packages/terra-clinical-data-grid/src/DataGrid.jsx

```diff
@@ -60,11 +60,12 @@ const propTypes = {
    */
   onRequestSectionCollapse: PropTypes.func,
   /**
-   * String that specifies the row height. Any valid CSS height value is accepted (i.e. 50px, 3rem, etc.)
+   * String that specifies the row height. Values are suggested to be in `rem`s (ex `'5rem'`), but any valid CSS height value is accepted.
+   * This value can be overridden by for a row by specifying a height on the given height.
```

Comment:
This should be rephrased. I think this was meant to read, “This value can be overridden by a height that is exclusively specified for a given row”?

Reply:
Good catch, fixed in cde2bd7c871a86afc4a36131a6f0bc94fa52cd5a